library/src/com/astuetz/PagerSlidingTabStrip.java

```diff
@@ -167,6 +163,8 @@ public PagerSlidingTabStrip(Context context, AttributeSet attrs, int defStyle) {
             tabTextColor = getColorStateList(textPrimaryColor);
         }
 
+        tabTextColorInactive = textPrimaryColor;
```

Comment:
I think `tabTextColorInactive` should have as default value the default `ColorStateList` of tabTextColor